### PR TITLE
Add/clarify SQL length limits

### PR DESCRIPTION
**Pull request by pushmeintheair**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
translate-docs-pr#15602

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/15602
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dashboard/dashboard-slow-query.md [hunk 1]
```diff
@@ -60,6 +60,10 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-slow-query/']
 
 ### SQL
 
+>**注意：**
+>
+>记录在`Query` 中的查询的长度受到 [`tidb_stmt_summary_max_sql_length`](/system-variables.md#tidb_stmt_summary_max_sql_length-new-in-v40)系统变量的限制。
```
Comment (hfxsd) [inline dashboard/dashboard-slow-query.md:65]:
```suggestion
>记录在 `Query` 中的查询的长度会受到 [`tidb_stmt_summary_max_sql_length`](/system-variables.md#tidb_stmt_summary_max_sql_length-从-v40-版本开始引入) 系统变量的限制。
```

File: system-variables.md [hunk 2]
```diff
@@ -4475,7 +4475,18 @@ Query OK, 0 rows affected, 1 warning (0.00 sec)
...
+
+-这个变量控制[statement summary tables](/statement-summary-tables.md)、[`SLOW_QUERY`](/information-schema/information-schema-slow-query.md)表和[TiDB Dashboard](/dashboard/dashboard-intro.md)中显示的SQL字符串长度，
+
+</CustomContent>
+
+<CustomContent platform="tidb-cloud">
+
+-这个变量控制[statement summary tables](/statement-summary-tables.md)和[`SLOW_QUERY`](/information-schema/information-schema-slow-query.md)表中显示的SQL字符串长度
+
+</CustomContent>
```
Comment (hfxsd) [inline system-variables.md:4489]:
```suggestion
- 这个变量控制 [Statement Summary Tables](/statement-summary-tables.md)、[`SLOW_QUERY`](/information-schema/information-schema-slow-query.md) 表和 [TiDB Dashboard](/dashboard/dashboard-intro.md) 中显示的 SQL 字符串长度。
```

Comment (pushmeintheair) [reply system-variables.md:4489]: 好的

File: system-variables.md [hunk 3]
```diff
@@ -4475,7 +4475,8 @@ Query OK, 0 rows affected, 1 warning (0.00 sec)
 - 类型：整数型
 - 默认值：`4096`
 - 范围：`[0, 2147483647]`
-- 这个变量控制 [statement summary tables](/statement-summary-tables.md) 显示的 SQL 字符串长度。
+
```
Comment (hfxsd) [inline system-variables.md:4478]:
```suggestion
```

File: dashboard/dashboard-slow-query.md [hunk 4]
```diff
@@ -60,6 +60,10 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-slow-query/']
 
 ### SQL
 
+>**注意：**
+>
+>记录在 `Query` 中的查询的长度会受到 [`tidb_stmt_summary_max_sql_length`](/system-variables.md#tidb_stmt_summary_max_sql_length-从-v40-版本开始引入) 系统变量的限制。
```
Comment (hfxsd) [inline dashboard/dashboard-slow-query.md:65]:
```suggestion
> 记录在 `Query` 中的查询的长度会受到 [`tidb_stmt_summary_max_sql_length`](/system-variables.md#tidb_stmt_summary_max_sql_length-从-v40-版本开始引入) 系统变量的限制。
```

**Review by hfxsd — APPROVED** [review]

**Review by hawkingrei — APPROVED** [review]

**Pull request merged**